File: app/src/shemerly/java/com/quran/labs/androidquran/data/QuranFileConstants.java
package com.quran.labs.androidquran.data;

import android.os.Build;

import com.quran.labs.androidquran.database.DatabaseHandler;
import com.quran.labs.androidquran.ui.util.TypefaceManager;

public class QuranFileConstants {
  // server urls
  public static final int FONT_TYPE = TypefaceManager.TYPE_UTHMANI_HAFS;

  // arabic database
  public static final String ARABIC_DATABASE =
      Build.VERSION.SDK_INT >= 21 ? "quran.ar.uthmani.v2.db" : "quran.ar.uthmani_simple.db";

  public static final String ARABIC_SHARE_TABLE =
      Build.VERSION.SDK_INT >= 21 ?
          DatabaseHandler.SHARE_TEXT_TABLE : DatabaseHandler.ARABIC_TEXT_TABLE;
  public static final boolean ARABIC_SHARE_TEXT_HAS_BASMALLAH = true;
  public static final boolean FETCH_QUARTER_NAMES_FROM_DATABASE = false;
}

File: app/src/shemerly/java/com/quran/labs/androidquran/data/QuranFileConstants.kt
package com.quran.labs.androidquran.data

import com.quran.labs.androidquran.database.DatabaseHandler
import com.quran.labs.androidquran.ui.util.TypefaceManager

object QuranFileConstants {
  // server urls
  const val FONT_TYPE = TypefaceManager.TYPE_UTHMANI_HAFS

  // arabic database
  const val ARABIC_DATABASE = "quran.ar.uthmani.v2.db"
  const val ARABIC_SHARE_TABLE = DatabaseHandler.SHARE_TEXT_TABLE
  const val ARABIC_SHARE_TEXT_HAS_BASMALLAH = true
  const val FETCH_QUARTER_NAMES_FROM_DATABASE = false

  const val FALLBACK_PAGE_TYPE = "shemerly"
}
